easybuild/easyconfigs/q/QIIME/QIIME-1.9.1.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Conda'
+
+name = 'QIIME'
+version = '1.9.1'
+
+homepage = 'http://qiime.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
+ from raw DNA sequencing data."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+requirements = "%(namelower)s=%(version)s matplotlib=1.4.3 mock nose"
+channels = ['bioconda']
```

Comment:
@boegel , I don't see anything in the [qiime recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/qiime/1.9.1/meta.yaml) that looks like it comes from conda-forge, but it won't hurt and it might be better to be consistent.